### DISCUSSION TITLE
Improve approval/rejection of events

### DISCUSSION THREAD

**Discussion by MrKrisKrisu** [Traewelling/traewelling#2002] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
Currently, the process of approving proposed events is a bottleneck as it requires admin approval. This slows down the process and creates a backlog.

**Describe the solution you'd like**
I propose implementing a crowdsourcing solution similar to Foursquare/Swarm. Users would receive a level of trustworthiness based on their past contributions, and proposed events would be visible to all users who can vote on whether to approve or reject them. This would speed up the process and involve the community in decision-making.

**Describe alternatives you've considered**

**Additional context**
This feature would improve the efficiency of the event approval process and allow for greater user participation in decision-making.

**Comment by jeyemwey** [comment]
I'm a little hesitant since those events will show up in each of the >3000 checkins that are happening every day, and need a high degree of quality to be helpful for our users. But I'm open for any experiments.

**Comment by MrKrisKrisu** [comment]
A new Role/Permission system was implemented in #2151. There is now also the role “event moderator”, where currently two trustworthy Träwelling users also have access to the corresponding backend interface.

Maybe we can expand something here in the future. I would definitely like to add model logging for events soon so that you can see who has changed what.
